.github/workflows/build.yml

```diff
@@ -119,8 +123,7 @@ jobs:
           env.RELEASE && matrix.ghc == '8.8.4'
         run: |
           .github/ci.sh build_cryptol
-          exe="$(cd deps/cryptol && cabal exec which cryptol)"
-          cp "$exe" "dist/bin"
+          .github/ci.sh extract_exe "cryptol" "dist/bin" "deps/cryptol/dist-newstyle"
```

Comment:
I'm wondering if this code could be generally refactored out completely... cabal install to a prefix directory?

Reply:
Cabal install, unfortunately, does not always [do the right thing](https://github.com/haskell/cabal/issues/6478). It needs to be `cabal build` and then running find in dist-newstyle. But here since we're building cryptol in a subdir, I have to do the weird thing there.

Ideally, saw-script relies on the latest hackage version of cryptol and the binary is downloaded from wherever we end up storing prebuilt binaries (probably just github releases for now). Then we won't need to do any of this and instead we can just download the appropriate binary.

Reply:
(that said. "cabal build + find in dist-newstyle + copy out to specified directory" is a good candidate for a helper function somewhere)